src/main/resources/assets/app/scripts/models/base_job.js

```diff
@@ -21,7 +21,6 @@ function($, Backbone, _, moment, BaseJobValidations) {
         encoded;
 
     encoded = _.map(args, function(arg) { return encodeURIComponent(arg); });
-    encoded.unshift('');
```

Comment:
It seems that is adding a fail-safe empty argument to the array, so if you have `[]`, it will work with `[""]`, which does not do any harm.